lib/menu.rb

```diff
@@ -0,0 +1,23 @@
+
+class Menu
+  attr_reader :menu_list
+
+  def initialize
```

Comment:
Might be worth having the menu as a constant? Unsure if it is more efficient in any way but it could be an alternative to make the initialize cleaner?